scripts/speech_recognition/convert_to_tarred_audio_dataset.py

```diff
@@ -473,7 +478,7 @@ def _create_shard(self, entries, target_dir, shard_id):
             base = base.replace('.', '_')
             squashed_filename = f'{base}{ext}'
             if squashed_filename not in count:
-                tar.add(entry['audio_filepath'], arcname=squashed_filename)
+                tar.add(args.dataset_dir+"/"+entry['audio_filepath'], arcname=squashed_filename)
```

Comment:
Remove the "/" part - that needs to be in the dataset dir, not automatically attached.